README.md

```diff
@@ -278,6 +278,8 @@ If you're using [**the Hardhat plugin**](https://github.com/0xSpaceShard/starkne
 
 ## Blocks
 
+Devnet starts with a genesis block (with a block number equal to 0).
```

Comment:
also add hash info

Reply:
Fixed in ebd8339e2576b5f0d94c108d360c95b15544bc58